build.rs

```diff
@@ -82,6 +83,8 @@ impl Version {
         }
         let patch = try!(num.parse::<u32>().map_err(|e| e.to_string()));
 
-        Ok(Version { major: major, minor: minor, patch: patch })
+        let stable = !parts[2].contains("-nightly") && !parts[2].contains("-beta");
```

Comment:
Please make sure all new lines introduced in this PR are wrapped to 79 columns (inclusive).